blogposts/2023-02-10-adding-snapshot-testing-to-rnw-apps.md

```diff
@@ -0,0 +1,93 @@
+---
+post_title: "Adding Jest Snapshot Testing to React Native Windows Apps"
+author1: chiaramooney
+post_slug: 2023-02-15-adding-snapshot-testing-to-rnw-apps
+post_date: 2023-02-15 12:00:00
```

Comment:
```suggestion
post_slug: 2023-03-02-adding-snapshot-testing-to-rnw-apps
post_date: 2023-03-02 12:00:00
```
let's set this date and merge so that it will get published then?